docs/_documentations/remote-removing.md

```diff
@@ -66,18 +64,19 @@ INFO[0001] Kubernetes namespace: CWCTL will not remove the namespace automatical
 use 'kubectl delete namespace mycodewind-ns' if you would like to remove it
 ```
 
->Since it is possible to install multiple Codewind servers into a Kubernetes namespace the cwctl will not remove the namespace as part of the uninstall. You should only remove the namespace if you no longer need it and are sure there are no other applications installed inside it. For now we will leave the namespace in place.
-
+Since it is possible to install multiple Codewind servers into a Kubernetes namespace, the `cwctl` command does not remove the namespace as part of the uninstall. You should only remove the namespace if you no longer need it and are sure there are no other applications installed inside it. 
```

Comment:
This line and the warning on line 98 sound similar. Is there a way to place this info in a single location?

Reply:
Rejigged the order of the content, and removed the duplication.